### PR TITLE
fix(stencil): enable weekly RC releases

### PR DESCRIPTION
**Pull request by pbr0ck3r**
Please read [CONTRIBUTING.md](CONTRIBUTING.md) for additional information on contributing to this repository!

## What this PR does / why we need it
- restencil

## Jira ID

[XX-XX]

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,16 +2,13 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@dev:test_release
+  shared: getoutreach/shared@dev:first
```
Comment (malept) [inline .circleci/config.yml:5]: suggestion:
```suggestion
  shared: getoutreach/shared@dev:2.27.0-rc.2
```

**Review by malept — CHANGES_REQUESTED** [review]
I think you need to set this value in `service.yaml` and restencil:

https://github.com/getoutreach/stencil-circleci/blob/972111352fff7723164358b5219eff964e482ab8/manifest.yaml#L32

**Comment by yj-yan** [comment]
A few more changes are needed in `service.yaml`: 
* Set`prereleasesBranch: main` in manifest.
* Set `autoPrereleases: true` to allow auto RC release.

**Comment by malept** [comment]
> A few more changes are needed in `service.yaml`:
> 
>     * Set`prereleasesBranch: main`.
> 
>     * Set `autoPrereleases: true` to allow auto RC release.
> 
>     * Use `rc` channel for `devbase` and `stencil-circleci`.

These instructions should go in the Jira issue.

**Review by malept — APPROVED** [review]

**Pull request merged**